config.sample

```diff
@@ -1,3 +0,0 @@
-u=user
-p=pass(API token recommended)
-s=site.domain.url
```

Comment:
Should not be deleting the sample from the repo. That's there to show others how to structure theirs.